core/connection/session_storage.go

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (C) 2018 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package connection
+
+import (
+	"github.com/mysteriumnetwork/node/client/stats"
+	"github.com/mysteriumnetwork/node/core/storage"
+	"github.com/mysteriumnetwork/node/session"
+)
+
+// SessionStorage describes functions for storing session objects
+type SessionStorage interface {
```

Comment:
We are agreed to put interfaces to called side, this time it is:
- `connection.Manager` just for `sessionSave` callback
- `sessionsEndpoint` just for `GetAll` callback

Reply:
- Interfaces are now in place.
- [`Save()`](https://github.com/mysteriumnetwork/node/pull/498/files#diff-328f448bf24fd1870fd2370b9c58c0b9R309)